tools/perf/components/perf_test_runner.py

```diff
@@ -291,12 +299,14 @@ def PrepareBinariesAndDirectories(configurations: List[RunnerConfig],
     out_dir = os.path.join(common_options.working_directory, description)
 
     binary_path = None
+    field_trial_config = None
 
     if common_options.do_run_tests:
       shutil.rmtree(out_dir, True)
       os.makedirs(out_dir)
       binary_path = PrepareBinary(out_dir, config.tag, config.location,
                                   config.browser_type)
+      assert (config.tag is not None)
```

Comment:
shouldn't it be `assert config.tag is not None`?

Reply:
Fixed